cmd/beacon/beacon.go

```diff
@@ -38,11 +39,11 @@ func init() {
 	// BeaconCmd.AddCommand(actions.VersionsCmd)
 	// BeaconCmd.AddCommand(actions.LogsCmd)
 	app, err := appcommands.GetNewApp("beacon", projectRunners.GetRunnerInstance,
-		appcommands.CommandDetails{Use: "create", DescShort: "Create marlin beacon", DescLong: "Create marlin beacon"},
+		appcommands.CommandDetails{Use: "create", DescShort: "Create marlin beacon", DescLong: "Create marlin beacon", AdditionalPreRunTest: keystore.KeystoreCheck},
```

Comment:
`PreRunTests` and `AdditionalPreRunTests` are supposed to identify correctly the runtime environment. Tasks which can be tested using this is
- Is supervisor installed?
- Is supervisor in running state?
etc. 
Any project specific tests which should run before creating the instances but is not "runtime check" and is "business logic test" should be hosted in runner's `PreRunSanity`.